Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/tomerfi/version-bumper:2.0.4
+FROM docker.io/tomerfi/version-bumper:2.1.0
```

Comment:
This was already done by dependabot here https://github.com/TomerFi/version-bumper-action/commit/343444a64c19c3bd8035df4570c45c4c45a33624.

You need to rebase your branch.